### PR TITLE
Landing page links and 404

### PR DESCRIPTION
**Pull request by NatasaKralj**
-Added a 404 page, when running locally can only be seen by navigating to http://localhost:1313/404.html
-Updated landing page links for products and most popular
-Added an alias redirect to landing page from old search page

### CODE REVIEW

File: layouts/404.html [hunk 1]
```diff
@@ -0,0 +1,18 @@
...
+        <br>
+        <br>
+        <br>
+        <br>
+        <br>
+        <div class="row">
+          <div class="col-12" style="text-align: center;">
+            <h1 id="title">404</h1>
+            <p>Oops! This page doesn't exist.</p>
+            <a href="{{ "/" | relURL }}" class="btn btn-outline-secondary home-btn">Back to Home</a>
```
Comment (MarkvanMents) [inline layouts/404.html:13]: Were we going to have an image on this one?
What do design say?

File: layouts/partials/landingpage/most-popular.html [hunk 2]
```diff
@@ -15,17 +15,17 @@ <h4 class="card-title">String Function Calls</h4>
...
 	<div class="col-12 col-sm-6 col-xl-4 image-card-col">
 		<div class="card image-card">
-		<img src="/images/most-popular-system-requirements.png" class="card-img-top" alt="...">
+		<img src="/images/most-popular-expressions.png" class="card-img-top" alt="...">
 		<div class="card-body">
-			<h4 class="card-title">System Requirements</h4>
+			<h4 class="card-title">Expressions</h4>
 			<br>
-			<a href="/refguide/system-requirements" class="btn btn-outline-secondary doc-link">Go to doc</a>
+			<a href="/refguide/expressions/" class="btn btn-outline-secondary doc-link">Go to doc</a>
```
Comment (MarkvanMents) [inline layouts/partials/landingpage/most-popular.html:28]: Is there a way to make the cards a fixed height? It looks a bit ugly when I don't use the whole screen width.
(LOW PRIORITY!)
image

File: layouts/partials/landingpage/product-cards.html [hunk 3]
```diff
@@ -1,70 +1,83 @@
...
 		<p class="card-text"><ul class="landing-list">
+		  <!-- Change links HERE -->
 		  <li class="link-list"><a href="/studio/general/">General Info</a></li>
 		  <li class="link-list"><a href="/studio/page-editor/">Pages</a></li>
 		  <li class="link-list"><a href="/studio-how-to/">Studio How-to's</a></li></ul></p>
 	  </div>
 	</div>
   </div>
   <div class="row justify-content-center inner-row">
+	<!-- Developer Portal card -->
```
Comment (MarkvanMents) [inline layouts/partials/landingpage/product-cards.html:30]: This is also a bit ugly. I'd rather have the icon on the same row as "Developer", I think. (LOW PRIORITY)
image

**Review by MarkvanMents — APPROVED** [review]
A couple of comments, but you can merge now if we can't do anything about them at present. They are low priority.

File: layouts/partials/landingpage/most-popular.html (hunk 2 above)
Comment (NatasaKralj) [reply layouts/partials/landingpage/most-popular.html:28]: I will add this to the to do list and see.

File: layouts/partials/landingpage/product-cards.html (hunk 3 above)
Comment (NatasaKralj) [reply layouts/partials/landingpage/product-cards.html:30]: I will add this to the to do list and see. I agree it's not very nice.

File: layouts/404.html (hunk 1 above)
Comment (NatasaKralj) [reply layouts/404.html:13]: The images already available didn't really seem suitable, we can go over them tomorrow.

**Pull request merged**